skate-plugin/artifactory-authenticator/src/main/kotlin/com/slack/intellij/artifactory/AuthPluginSettings.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.intellij.artifactory
+
+import com.intellij.ide.plugins.auth.PluginRepositoryAuthListener
+import com.intellij.openapi.components.BaseState
+import com.intellij.openapi.components.Service
+import com.intellij.openapi.components.SimplePersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+
+@Service(Service.Level.APP)
```

Comment:
I know for the skate plugin settings we used `Service.Level.PROJECT`. is there a difference between that and APP?

Reply:
Yep, plug-in repositories aren't project specific so we use app scope (I.e. IDE-wide)